src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,367 @@
+---
+title: Payments in Flex
+slug: payments-overview
+updated: 2021-11-03
+category: background
+ingress:
+  This article introduces how payments work in Flex in general, and
+  describes the default Stripe payment gateway integration
+published: false
+---
+
+Flex is a full-fledged marketplace solution, complete with payment
+capabilities. In this article, you will learn about the Flex default
+payment integration, implemented with Stripe, as well as the
+alternatives in case you decide to not use Stripe on your marketplace.
+
+## Marketplace payment flow
+
+In a nutshell, a basic payment flow in a marketplace contains five
+significant steps:
+
+![Payment gateway logic](./payment-flow.png)
```

Comment:
This section is now almost duplicate with the 3rd party payment gateway article beginning. Any feedback welcome re: should one of the duplicates be removed or trimmed somehow!